opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -196,11 +269,25 @@ mod tests {
             .attributes
             .clone()
             .expect("Attributes are expected");
+        #[cfg(not(feature = "experimental_metadata_attributes"))]
         assert_eq!(attributes.len(), 4);
+        #[cfg(feature = "experimental_metadata_attributes")]
+        assert_eq!(attributes.len(), 9);
         assert!(attributes.contains(&(Key::new("name"), "my-event-name".into())));
         assert!(attributes.contains(&(Key::new("event_id"), 20.into())));
         assert!(attributes.contains(&(Key::new("user_name"), "otel".into())));
         assert!(attributes.contains(&(Key::new("user_email"), "otel@opentelemetry.io".into())));
+        #[cfg(feature = "experimental_metadata_attributes")]
+        {
+            assert!(attributes.contains(&(Key::new("log.source.file.name"), "layer.rs".into())));
+            assert!(attributes.contains(&(
+                Key::new("log.module.path"),
+                "opentelemetry_appender_tracing::layer::tests".into()
+            )));
+            // The other 3 experimental_metadata_attributes are too unstable to check statically.
+            // Ex.: The path will be different on a Windows and Linux machine.
```

Comment:
nit: we can still check if an attribute with that key is added, without validating the actual value? We are generally very low in code coverage, so trying to making sure new code is always supported by tests!

Reply:
Will do it.

Reply:
Done.